pkgs/tools/security/gitjacker/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "gitjacker";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "liamg";
+    repo = "gitjacker";
+    rev = "v${version}";
+    sha256 = "0fg95i2y8sj7dsvqj8mx0k5pps7d0h1i4a3lk85l8jjab4kxx8h9";
+  };
+
+  vendorSha256 = null;
+
+  doCheck = false;
```

Comment:
Please add a comment why you disabled checkPhase.

Reply:
Can remember why I didn't enable the tests. They are now.